README.md

```diff
@@ -270,19 +268,9 @@ The elements can be specified using [FHIRPath](http://hl7.org/fhirpath/) syntax.
 |redact|All elements| Removes the element. See the parameters section below to handle special cases.|
 |dateShift|Elements of type date, dateTime, and instant | Shifts the value using the [Date-shift algorithm](#date-shift-algorithm).
 
-### Type Rules
-Type rules are key-value pairs that can be used to specify the de-identification methods at the datatype level. Ex:
-
-```json
-"date": "dateShift"
-```
-The datatypes can be any of the [FHIR datatypes](https://www.hl7.org/fhir/datatypes.html). The method can be one from the following table.
-
-|Method| Applicable to | Description
-| ----- | ----- | ----- |
-|keep|All datatypes | Retains the value as is. |
-|redact|All datatypes| Removes the element. See the parameters section below to handle special cases. |
-|dateShift|date, dateTime, and instant datatypes | Shifts the value using the [Date-shift algorithm](#date-shift-algorithm).
+2 extension methods can be used in FHIR path rule to simplify the FHIR path:
+- nodesByType('_typename_'): return descendants of type '_typename_', nodes in bundle resource and contained list would be excluded. 
+- nodesByName('_name_'): return descendants of node name '_name_', nodes in bundle resource and contained list would be excluded. 
 
 ### Parameters
 Parameters affect the de-identification methods specified in the type rules and path rules. 

```

Comment:
There are no type riles and path rules any more

Reply:
Removed path rule and type rule.